airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceStandardTest.java

```diff
@@ -142,13 +145,30 @@ public void setup() throws Exception {
       createSchemas();
     }
     database.execute(connection -> {
-
       connection.createStatement()
-          .execute(String.format("CREATE TABLE %s(id INTEGER, name VARCHAR(200), updated_at DATE);", getFullyQualifiedTableName(TABLE_NAME)));
+          .execute(String.format("CREATE TABLE %s(id INTEGER, name VARCHAR(200), updated_at DATE, PRIMARY KEY (id));",
+              getFullyQualifiedTableName(TABLE_NAME)));
       connection.createStatement().execute(
           String.format(
               "INSERT INTO %s(id, name, updated_at) VALUES (1,'picard', '2004-10-19'),  (2, 'crusher', '2005-10-19'), (3, 'vash', '2006-10-19');",
               getFullyQualifiedTableName(TABLE_NAME)));
+
```

Comment:
good call adding these test cases here. glad that this could be added without having to rewrite all of the tests!